wgsl/index.bs

```diff
@@ -5483,10 +5483,10 @@ See [[#function-call-statement]].
   <tr algorithm="variable reference">
        <td>
           |v| is an [=identifier=] [=resolves|resolving=] to
-          an [=in scope|in-scope=] variable declared in [=address space=] |S|
-          with [=store type=] |T|
+          an [=in scope|in-scope=] variable declared in [=address space=] |AS|
+          with [=store type=] |T| and address mode |AM|
```

Comment:
Is this `address mode |AM|` or `[=access mode=] |AM|` ?

Reply:
heh.  Access mode.  :-)